internal/backend/integrations/integrations.go

```diff
@@ -18,12 +18,22 @@ import (
 	"go.autokitteh.dev/autokitteh/integrations/scheduler"
 	"go.autokitteh.dev/autokitteh/integrations/slack"
 	"go.autokitteh.dev/autokitteh/integrations/twilio"
+	"go.autokitteh.dev/autokitteh/internal/backend/configset"
 	"go.autokitteh.dev/autokitteh/sdk/sdkintegrations"
 	"go.autokitteh.dev/autokitteh/sdk/sdkservices"
 )
 
-func New(s sdkservices.Secrets) sdkservices.Integrations {
-	return sdkintegrations.New([]sdkservices.Integration{
+type Config struct {
+	Test bool `koanf:"test"`
```

Comment:
Is `Test` the right name?
Also maybe instead of a configuration this can be guarded by a build tag?

Reply:
there is a single binary for all modes, so this can't be a build tag. i don't have a better idea than `Test`... But it jives nicely with test mode.